packages/react-ui-components/src/components/SchemaForm/SchemaForm.tsx

```diff
@@ -113,7 +115,6 @@ export function KvSchemaForm<T, S extends StrictRJSFSchema = RJSFSchema>({
 	};
 
 	const discardChanges = () => {
-		setFormData(cloneDeep(submittedData) as T);
```

Comment:
the new values from discarded option will come from the parent?

Reply:
Yap, that's the point, no internal states.